R/run_trisk.R

```diff
@@ -163,7 +167,7 @@ globalVariables(c(names(formals(run_trisk)), "iter_var", "end_year"))
 read_and_process_and_calc <- function(args_list) {
   list2env(args_list, envir = rlang::current_env())
 
-  log_path <- file.path(output_path, paste0("log_file_", iter_var, ".txt"))
+  log_path <- file.path(output_path, paste0("log_file_", iter_var, sep = "_", shock_scenario, sep = "_", scenario_geography, ".txt"))
```

Comment:
I think it's fine here, but when a name grows too long and uses mixed seperators, string formatting is an option
```r
sprintf("log_file_%s_%s_%s.txt", iter_var, shock_scenario, scenario_geography)
```

Reply:
thanks